packages/cli/src/commands/bundle/assetPathUtils.js

```diff
@@ -33,7 +33,7 @@ function getAndroidAssetSuffix(scale: number): string {
     case 4:
       return 'xxxhdpi';
     default:
-      throw new Error('no such scale');
+      return '';
```

Comment:
Do we handle that error somewhere else? Or in other words, what is the expected action here?

Reply:
Yes, the error is being handled on [#L57](https://github.com/react-native-community/cli/blob/d82a457509e1ca8fab08a15afec130e6773d7417/packages/cli/src/commands/bundle/assetPathUtils.js#L57). Before making this change, that was error was never being hit because we were throwing here. I couldn't use undefined/null because `flow` gave errors then so I had to fallback to empty string.